src/share/poudriere/jail.sh

```diff
@@ -117,8 +117,8 @@ list_jail() {
 			_jget method ${name} method
 			_jget mnt ${name} mnt
 			_jget timestamp ${name} timestamp || :
-			if [ -r "${mnt}/sys/sys/param.h" ]; then
-				osversion=$(awk '/^\#define[[:blank:]]__FreeBSD_version/ {print $3}' "${mnt}/sys/sys/param.h")
+			if [ -r "${mnt}/usr/src/sys/sys/param.h" ]; then
+				osversion=$(awk '/^\#define[[:blank:]]__FreeBSD_version/ {print $3}' "${mnt}/usr/src/sys/sys/param.h")
```

Comment:
I think it would be safer to check /usr/include/sys/param.h as /usr/src is not required to build much so is often excluded.